### PR TITLE
Feature - extraction report updates

### PR DESCRIPTION
**Pull request by rkm**
## Proposed Changes

Updates the extraction report to be more human-readable

## Types of changes

What types of changes does your code introduce? Tick all that apply.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation-Only Update (if none of the other choices apply)
  - In this case, ensure that the message of the head commit from the source branch is prefixed with `[skip ci]`

## Checklist

By opening this PR, I confirm that I have:

- [x] Reviewed the [contributing](https://github.com/SMI/SmiServices/blob/master/CONTRIBUTING.md) guidelines for this repository
- [x] Ensured that the PR branch is in sync with the target branch (i.e. it is automatically merge-able)
- [x] Updated any relevant API documentation
- [x] Created or updated any tests if relevant
- [x] Accurately updated the [CHANGELOG](https://github.com/SMI/SmiServices/blob/master/CHANGELOG.md)
  - NOTE: This ***must*** include any changes to any of the following files: default.yaml, any of the RabbitMQ server configurations, GlobalOptions.cs
- [x] Listed myself in the [CONTRIBUTORS](https://github.com/SMI/SmiServices/blob/master/CONTRIBUTORS.md) file 🚀
- [x] Requested a review by one of the repository maintainers

## Issues

- Closes https://dev.azure.com/SmiOps/MVP%20Service/_workitems/edit/160/

### CODE REVIEW

**Comment by rkm** [comment]
LGTM, but the build is failing for some RabbitMQ-y reason.

**Comment by jas88** [comment]
I'm partly glad about that - I put the RabbitMQ update on hold because of similar issues, but now we're seeing it in the existing version too ...

**Comment by rkm** [comment]
Can you tell what the issue is? I think the only variable is the RMQ client version that gets pulled from NuGet?

**Comment by jas88** [comment]
Various odd manifestations, depending on which version and which issues I'd worked around already; the one you saw on Travis here came up quite recently - the unit test helped in tests/common/Smi.Common.Tests/DeadLetterMessagingTests/DeadLetterTestHelper.cs always declares exchanges as "direct" when the control message ones need to be "topic", so I worked around that in the dependabot/RabbitMQ-6 feature branch.

File: src/microservices/Microservices.CohortPackager/Execution/ExtractJobStorage/MongoDB/MongoExtractJobStore.cs [hunk 1]
```diff
@@ -311,8 +311,11 @@ protected override ExtractJobInfo GetCompletedJobInfoImpl(Guid jobId)
             IAsyncCursor<MongoExpectedFilesDoc> cursor = _completedExpectedFilesCollection.FindSync(Builders<MongoExpectedFilesDoc>.Filter.Eq(x => x.Header.ExtractionJobIdentifier, jobId));
             while (cursor.MoveNext())
                 foreach (MongoExpectedFilesDoc expectedFilesDoc in cursor.Current)
+                {
+                    yield return new Tuple<string, int>(expectedFilesDoc.Key, -1);
```
[the remark this answers is not in the document]
Comment (rkm) [reply src/microservices/Microservices.CohortPackager/Execution/ExtractJobStorage/MongoDB/MongoExtractJobStore.cs]: Have fixed this up now.

**Pull request merged**